plan/testdata/select_from_value_scalar.json

```diff
@@ -0,0 +1,152 @@
+                                    "value": {
+                                      "literal": {
+                                        "i32": 1
+                                      }
+                                    }
+                                  },
+                                  {
+                                    "value": {
+                                      "literal": {
+                                        "i32": 1
+                                      }
+                                    }
+                                  }
+                                ],
+                                "functionReference": 1,
+                                "outputType": {
+                                  "i32": {
+                                    "nullability": "NULLABILITY_REQUIRED"
+                                  }
+                                }
+                              }
+                            },
+                            {
+                              "scalarFunction": {
+                                "arguments": [
+                                  {
+                                    "value": {
+                                      "literal": {
+                                        "i32": 2
+                                      }
+                                    }
+                                  },
+                                  {
+                                    "value": {
+                                      "literal": {
+                                        "i32": 2
+                                      }
+                                    }
+                                  }
+                                ],
+                                "functionReference": 1,
+                                "outputType": {
+                                  "i32": {
+                                    "nullability": "NULLABILITY_REQUIRED"
+                                  }
+                                }
+                              }
+                            }
+                          ]
+                        }
+                      ]
+                    }
+                  }
+                },
+            "expressions": [
+              {
+                "selection": {
+                  "directReference": {
+                    "structField": {}
+                  },
+                  "rootReference": {}
+                }
+              },
+              {
+                "selection": {
+                  "directReference": {
+                    "structField": {
+                      "field": 1
+                    }
+                  },
+                  "rootReference": {
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "names": [
+          "col0",
+          "col1",
+          "col0",
+          "col1"
+        ]
+      }
+    }
+  ],
+  "version": {
```

Comment:
If we clear this part of the plan before comparing we won't need to update it when the version changes.

Reply:
I think this would need a discussion. Will take it in separate PR.